test/functional/api/base_controller_subclass_test.rb

```diff
@@ -221,32 +221,51 @@ class Api::TestableControllerTest < ActionController::TestCase
         @testable_scope2.stubs(:merge).returns(@testable_scope1)
         @child_associacion.stubs(:merge).returns(@testable_scope1)
         @testable_scope1.stubs(:readonly).returns(@testable_scope1)
-        @testable_scope1.expects(:find).with('1').returns(@testable_obj)
-        @testable_scope1.expects(:empty?).returns(false)
         Testable.stubs(:joins).returns(@child_associacion)
       end
 
-      it 'should return nested resource for unauthorized resource' do
-        Testable.stubs(:where).returns(@testable_scope2)
-        Testable.stubs(:scoped).returns(@testable_scope2)
+      context 'aaa' do
+        setup do
+          @testable_scope1.expects(:find).with('1').returns(@testable_obj)
+          @testable_scope1.expects(:empty?).returns(false)
+        end
 
-        get :nested_values, :domain_id => 1, :id => 1
+        it 'should not return nested resource for unauthorized resource' do
```

Comment:
I don't think the word "not" should be added here, because it _does_ return a nested resource scope, but without using .authorized.